sysboundary/sysboundarycondition.cpp

```diff
@@ -785,48 +785,60 @@ namespace SBC {
       return flowtoCellsBlock;
    }
    
-   Real SysBoundaryCondition::fieldBoundaryCopyFromExistingFaceNbrMagneticField(
+   Real SysBoundaryCondition::fieldBoundaryCopyFromSolvingNbrMagneticField(
       FsGrid< std::array<Real, fsgrids::bfield::N_BFIELD>, 2> & perBGrid,
       FsGrid< fsgrids::technical, 2> & technicalGrid,
       cint i,
       cint j,
       cint k,
-      cuint component
+      cuint component,
+      cuint mask
```

Comment:
Since mask depends directly on component we don't need it as an input parameter, it can be extracted in this function.

Reply:
I'll look that up. Possibly. But we have more than B in compute::, so we have to be careful.